templates/Web/Projects/AddPackageJsonDefault/README.md

```diff
@@ -9,15 +9,28 @@ In the root directory of the project...
 
 ### Deployment
 
-The generated templates can be deployed to Azure App Service using the following steps:
+Follow these steps to deploy to Azure App Service if you selected Azure App Service when creating your project:
 
-1. In the root directory of the project `yarn build` or `npm build` to create a build folder.
-2. Move the build folder inside the server folder.
-3. Deploy the server folder to Azure App Service using the [Azure App Service Extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice).
-4. If a database is used, add the environment variables defined in .env to your Application Settings.
-5. Consider adding authentication and securing back-end API's by following [Azure App Service Security](https://docs.microsoft.com/en-us/azure/app-service/overview-security).
+1. Press `Ctrl + Shift + P` in Windows/Linux or `Shift ⇧ + Command ⌘ + P` in Mac and type in "Web Template Studio: Deploy App" to start deploying your app.
+2. Once the deployment is done, click "Browse website" in the notification window on the lower right corner to check out your newly deployed app.
+3. Consider adding authentication and securing back-end API's by following [Azure App Service Security](https://docs.microsoft.com/en-us/azure/app-service/overview-security).
    Full documentation for deployment to Azure App Service can be found here: [Deployment Docs](https://github.com/Microsoft/WebTemplateStudio/blob/dev/docs/deployment.md).
 
+If you did not select Azure App Service and would like to create a new Azure App Service web app, follow these steps:
+
+1. Press `Ctrl + Shift + P` in Windows/Linux or `Shift ⇧ + Command ⌘ + P` in Mac and type in "Azure App Service: Create New Web App..." to create a new web app. <br>
+   - Select your subscription
+   - Enter your web app name
+   - Select Linux as your OS
+   - Select Node.js 10.14 for a Node/Express application, Python 3.7 for a Flask application
+2. Press `Ctrl + Shift + P` in Windows/Linux or `Shift ⇧ + Command ⌘ + P` in Mac and type in "Web Template Studio: Deploy App" to start deploying your app.
+   - You will have to select the same subscription again
+   - Find the web app that you've just created
+   - Click yes in the notification windows on the lower right corner (build prompt)
+   - Click "Deploy" on the windows pop-up (deploy window)
+   - Click "Yes" on the notification windows on the lower right corner (default prompt)
```

Comment:
![image](https://user-images.githubusercontent.com/20333350/62968998-2181c000-bdc1-11e9-929f-a3bf75f171f6.png)

Reply:
What if we want to deploy a different application to a different resource group? Would it still give us an option to do that?

Reply:
You'll need to remove the settings.json in the .vscode folder. What do you think? Should we add it to the instruction?

Reply:
notification window (singular)